docs/src/documentation/03-components/09-text/text/01-guidelines.mdx

```diff
@@ -1,3 +1,105 @@
 ---
 title: Guidelines
 ---
+
+<ReactExample name="Text" />
+
+See also the general guidelines in [Typography](https://orbit.kiwi/foundation/typography/).
+
+## When to use
+
+- To communicate information.
+- To control the visual styling of text.
+- Only sparingly to add necessary information with a clear purpose.
+
+### When not to use
+
+- To add structure and divide screens into sections---use a [heading](/components/text/heading/).
+
+## Component status
+
+<ComponentStatus component="Text" />
+
+## Content structure
+
+<FigmaFile
+  file="4QJZqvBvRrLu6t9mwObCkA"
+  node="210%3A0"
+  alt="Size: distinguishes the importance of the text and is the primary way to create a hierarchy; type: distinguishes the purpose of the text and enables a hierarchy without sacrificing readability."
+/>
+
+## Look & feel
+
+### Base everything off normal primary
+
+Start with normal-sized, primary, unweighted text.
+Then add importance first with size and then weight.
+Show less important things first with smaller text and then with secondary color.
+
+### Keep all text in a block one color
+
+Inside a paragraph, all the text should be the same color.
+To emphasize something, use a stronger weight.
+
+<Guidelines type="do" title="Use one color">
+
+<FigmaFile
+  file="fannvRpkOJK6uxxT33EKaa"
+  node="56%3A2"
+  alt="A sentence with two words in bold indicating emphasis."
+/>
+<FigmaFile
+  file="fannvRpkOJK6uxxT33EKaa"
+  node="56%3A1"
+  alt="A sentence with words in green and red to indicate emphasis."
+/>
+
+</Guidelines>
+
+### Text size
+
+The basic styling options for text are the three possible sizes
+(<InlineToken value="fontSizeTextSmall" alternateName />,
+
+<InlineToken value="fontSizeTextNormal" alternateName />, and <InlineToken
+  value="fontSizeTextLarge"
+  alternateName
+/>
+). These sizes should give you a clear hierarchy for all your text.
```

Comment:
If this was caused by Prettier, we should remember to ignore it here. Maybe adding a temporary `<!-- prettier-ignore -->` should work. 🤷 https://prettier.io/docs/en/ignore.html#html

Reply:
Using ESLint disabling instead, but yes, seems to have been caused by Prettier.

Reply:
And neither seemed necessary when they were put so no line started with `<`